text/0000-pack-suggest-stacks.md

```diff
@@ -0,0 +1,81 @@
+# Meta
+[meta]: #meta
+- Name: Suggest Stacks
+- Start Date: 2019-04-25
+- CNB Pull Request: (leave blank)
+- CNB Issue: #buildpack/pack#156
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+A `suggest-stacks` command will be added to the `pack` CLI help buildpack authors discover stacks. The output of this command will include the `io.buildpacks.stack.bionic` stack (maintained by the Cloud Native Buildpacks project) and other stacks that are popular within the community.
```

Comment:
could this be `pack stacks`?

Reply:
`pack suggest-stacks` would be consistent with the existing `pack suggest-builders` command. I think the idea behind the `suggest-*` command names is to convey to users that these lists are not comprehensive. If for example, I create a stack, users should not expect it to appear in this list.